fab/pillows/swiss.yml

```diff
@@ -5,3 +5,5 @@ exclude_pillows:
   - ReportCaseToElasticsearchPillow
   - ReportXFormToElasticsearchPillow
   - CaseSearchToElasticsearchPillow
+  - kafka-ucr-static
+  - kafka-ucr-main
```

Comment:
didn't see them using any ucr so figured this was fine. @czue let me know if you think that's an issue